spec/index.html

```diff
@@ -248,5 +249,155 @@ <h2>Basic Concepts</h2>
+              useful documentation at that URI. For more information and background
+              please refer to [[RFC6906]].</p>
+            <p>This specification defines seven values for the <code>profile</code> parameter.</p>
+            <dl>
+              <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-expanded-document-form">expanded YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-compacted-document-form">compacted YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#context</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11#dfn-context-document">YAML-LD context document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-flattened-document-form">flattened YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#frame</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11-framing#dfn-frame">YAML-LD frame document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-framed-document-form">framed YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#extended</code></dt>
+              <dd>To request or specify <a data-lt="extended document form">extended YAML-LD document form</a>.
+                <div class="ednote">
+                  This is a placeholder for specifying something like an
+                  <dfn data-lt="extended-document-form">extended YAML-LD document form</dfn>
+                  making use of YAML-specific features.
+                </div></dd>
+            </dl>
+            <p>All other URIs starting with <code>http://www.w3.org/ns/json-ld</code>
+              are reserved for future use by JSON-LD specifications.</p>
+            <!--p>Other specifications MAY create further structured subtypes
+              by using `+ld+json` as a suffix for a new base subtype, as in
+              `application/example+ld+json`.
+              Unless defined otherwise, such subtypes use the same
+              fragment identifier behavior as `application/ld+json`.</p-->
+            <p>Other specifications may publish additional `profile` parameter
+              URIs with their own defined semantics.
+              This includes the ability to associate a file extension with a `profile` parameter.</p>
+            <p>
+              When used as a <a data-cite="RFC4288#section-4.3">media type parameter</a> [[RFC4288]]
+              in an <a data-cite="rfc7231#rfc.section.5.3.2">HTTP Accept header</a> [[RFC7231]],
```

Comment:
```suggestion
              in an <a data-cite="rfc9110#rfc.section.12.5.1">HTTP Accept header field</a> [[RFC9110]],
```

Probably, we could simplify this section, eg.

> When using this parameter in content negotiation (see Section 12.5 of RFC9110), its value MUST be
> enclosed in double quotes...

Q: is the procedure above specific to the profile parameter or it is generally valid for all media type parameters? In case, I can double check.

Reply:
I believe it's generally true about parameter values.